src/components/Book.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = ({ book, handleRemove }) => (
+  <tr>
+    <td>{book.id}</td>
+    <td>{book.title}</td>
+    <td>{book.category}</td>
+    <td>
+      <button
+        onClick={() => handleRemove(book)}
+        type="button"
+      >
+        delete
+      </button>
+    </td>
+  </tr>
+);
+
+Book.propTypes = {
+  book: PropTypes.shape.isRequired,
```

Comment:
[Optional] With PropTypes.shape we should define an actual shape for the object.

Reply:
Thanks you very much.
@FabienNeibaf made that change in the next milestone[ pull request](https://github.com/FabienNeibaf/Bookstore/pull/3/commits/91294a3d1ed705ddc6c371f0b06d3cb7eef48241)